cmake/BLTPrivateMacros.cmake

```diff
@@ -406,10 +406,12 @@ macro(blt_setup_cuda_target)
         # instead of leaving it up to CMake to decide
         # Note: we don't do this when depending on just 'cuda_runtime'
         set(_cuda_sources)
-        set(_non_cuda_sources)
+        set(_non_cuda_fortran_sources)
+        set(_non_cuda_python_sources)
         blt_split_source_list_by_language(SOURCES      ${arg_SOURCES}
                                           C_LIST       _cuda_sources
-                                          Fortran_LIST _non_cuda_sources)
+                                          Fortran_LIST _non_cuda_fortran_sources
+                                          Python_LIST  _non_cuda_python_sources)
```

Comment:
Python will never be in the sources list at the point because this is for compiling executables.  These changes can be reverted.

Reply:
I reverted those changes and the corresponding changes in `blt_add_hip_library` and `blt_add_hip_executable` for similar reasons. Those changes are no longer in amended commit [f41fe920](https://github.com/LLNL/blt/pull/412/commits/f41fe9201491b7b98eb61f76e8570188c3db9d90), which is supposed to contain all of the `blt_split_source_by_language`-related changes prior to adding `blt_add_yapf_target`.